cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -1374,8 +1380,20 @@ public void run() {
         if (_executionTimerInvolveBrokerRemovalOrDemotion != null) {
           _executionTimerInvolveBrokerRemovalOrDemotion.update(duration, TimeUnit.MILLISECONDS);
         }
+
+        String executionStatusString = String.format(" task Id: %s; removed brokers: %s; demoted brokers: %s; total time used: %dms.",
+                                               _uuid,
+                                               _removedBrokers,
+                                               _demotedBrokers,
+                                               duration
+                                               );
+
+        if (_executionException != null) {
+          LOG.info("Execution failed: {}. Exception: {}", executionStatusString, _executionException.getMessage());
+        } else {
+          LOG.info("Execution finished: {}. ", executionStatusString);
```

Comment:
For consistency (or better consistency, anyway) with the other logging, could you s/finished/succeeded/  ?  "Finished" is a little bit ambiguous.